Dockerfile

```diff
@@ -23,7 +37,7 @@ RUN export MAKEFLAGS=' -j8' &&  cd /gentle/ext && \
 
 ADD . /gentle
 RUN cd /gentle && ./install_models.sh
-RUN cd /gentle && python3 setup.py develop
+RUN cd /gentle && pip3 install redis twisted && python3 setup.py develop
```

Comment:
The setup file already has

```
install_requires=['twisted', 'redis', 'python-json-logger'],
```

do we need this here too?

Reply:
No, I removed it. That was a workaround to the broken "pip search" needed implicitly by setuptools on Python 3.5. I just moved everything onto Python 3.9.